ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-ref.yaml

```diff
@@ -17,5 +17,17 @@ ref:
       default:  "no"
       documentation: |-
         This field represents the VPC would be a disconnected network (without NAT gateway), or not.
+    - name: NUMBER_SUBNETS_PAIR_PER_ZONE
```

Comment:
in case [OCP-77281 - [IPI-on-IBMCloud] create a cluster with more 15 subnets](https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-77281) will create 16 subnets,  install config expect prompt error. 
[OCPBUGS-43940](https://issues.redhat.com/browse/OCPBUGS-43940) [IBMCloud] install cluster with >15 subnets got validation_failed_max_items error

Reply:
This PR does not care about the bug, the PR is making the script have the capacity to creating a large number of subnets, what value is set totally depends on users.